### PR TITLE
EMCAL DCS DP processing

### PR DESCRIPTION
**Pull request by mpoghos**

### CODE REVIEW

**Review by shahor02 — DISMISSED** [review]
conditionally approving to trigger CI

File: Detectors/EMCAL/calib/src/EMCDCSProcessor.cxx
Comment (mfasDa) [inline Detectors/EMCAL/calib/src/EMCDCSProcessor.cxx]: Misformated. Also lines below. Please run clang-format -I on that file.

**Review by mfasDa — CHANGES_REQUESTED** [review]
Several files are not properly formatted - please run `clang-format -i ...` on the file. Also please change the commit message header to 

```
[EMCAL-556] EMCAL DCA DP processing
```

and specify in the message body which datapoints are implemented - I want to see the commit message in this format before approval.

The code has no documentation, even very few comments, which is not helpful but for the moment not a blocker from approval.

Otherwise the code looks good I would say.

**Review by mfasDa — DISMISSED** [review]
Looks good now!

**Review by mfasDa — DISMISSED** [review]

**Review by shahor02 — APPROVED** [review]
conditionally approving to restart CI

**Comment by mfasDa** [comment]
@shahor02 Is the approval mandatory to start the CI? In this case, is the approval from me as codeowner sufficient?

**Comment by shahor02** [comment]
Hi @mfasDa 
Approval is needed for the 1st 1st-timer commit. I guess the codeowner approval should work, you can try if you find another 1st-timer

**Comment by mpoghos** [comment]
Hi @shahor02 and @mfasDa 
Do you understand why the build/AliceO2/O2/o2/macOS test filed? I think nothing related to my commit. is this a blocker for merging?

**Comment by shahor02** [comment]
yes, it is unrelated

**Comment by mfasDa** [comment]
@shahor02 Thanks for the reply! I already did the approval at the very first stage so it seems the code owner approval is not sufficient.

**Pull request merged**
